### ISSUE TITLE
Show a Toast on Copy button too

### ISSUE THREAD

**Issue by XhmikosR** [jakearchibald/svgomg#319]
While using <kbd>Ctrl+C</kbd> shows a Toast success message, when using the floating copy button there's no feedback.

It would probably make sense to streamline this.

**Comment by BarokDG** [comment]
Can I help with this? It doesn't look like it's implemented.

**Comment by XhmikosR** [comment]
Since it's not implemented, generally, there's no need to ask, just go ahead and propose a PR :)

**Comment by BarokDG** [comment]
Okay, my bad. Thanks!

I've been working on it for a while today but I can't find a good solution. The CopyButton class doesn't have access to the toastsUi since the toastsUi is initialised inside main-controller.js. I can try to create a tooltip instead tho. What do you think?

**Comment by XhmikosR** [comment]
Yeah, that was my problem when I had a quick look at it. Not sure if it's worth changing too much code to support this, but it's an inconsistency nonetheless.

@jakearchibald WDYT?
